tests/src/kakarot/precompiles/test_precompiles.py

```diff
@@ -10,6 +12,7 @@ class TestRun:
                 (0x6, "Kakarot: NotImplementedPrecompile 6"),
                 (0x7, "Kakarot: NotImplementedPrecompile 7"),
                 (0x8, "Kakarot: NotImplementedPrecompile 8"),
+                (0x0A, "Kakarot: NotImplementedPrecompile :"),
```

Comment:
plz update the Error.notImplementedPrecomile(address) to handle 10 (here 10 becomes `:` because in ASCII, after `9` it's `:`)

Reply:
added a special handling of case == 10 (only precompile w/ two digits)

Reply:
added a special handling of case == 10 (only precompile w/ two digits)